doc/examples/segmentation/plot_multiotsu.py

```diff
@@ -0,0 +1,76 @@
+"""
+=======================
+Multi-Otsu Thresholding
+=======================
+
+In this example we use multi-Otsu threshold to separate the pixels of an
+input image into three different classes, each one obtained according to
+the intensity of the gray levels within the image.
+
+Multi-Otsu calculates several thresholds, determined by the number of desired
+classes. The default number of classes is 3: for obtaining three classes, the
+algorithm returns two threshold values. They are represented by a red line in
+the histogram below.
+
+.. [1] Liao, P-S. and Chung, P-C., "A fast algorithm for multilevel
+thresholding", Journal of Information Science and Engineering 17 (5):
+713-727, 2001.
+"""
+
+from skimage.data import camera
+from skimage.filters import threshold_multiotsu
+
+import matplotlib
+import matplotlib.pyplot as plt
+
+
+# Stablishing the font size for all plots.
+matplotlib.rcParams['font.size'] = 9
+
+# The input image.
+image = camera()
+
+# Applying multi-Otsu threshold for the default value, generating
+# three classes.
+thresh = threshold_multiotsu(image)
+
+# Using the values on thresh, we generate the three regions.
+region1 = image <= thresh[0]
+region2 = (image > thresh[0]) & (image <= thresh[1])
+region3 = image > thresh[1]
```

Comment:
A future feature will replace that with a function `apply_threshold`, but we will keep that for this PR.